setup.py

```diff
@@ -116,6 +116,9 @@
             'yapf',
         ],
         'horovod': ['horovod[pytorch]>=0.19'],
+        'funsor': [
+            'funsor[torch] @ git+git://github.com/pyro-ppl/funsor.git@master',  # TODO update to release branch
+        ],
```

Comment:
It would be a little safer to pin to either a commit or a release. I'm worried that if we pin to a moving branch and that branch changes then we wouldn't be able to git bisect. What do you think of any of these alternatives:
- pinning to funsor 0.2.0
- releasing current master and pinning to 0.2.1
- creating a tag in the funsor repo named say `pyro-1.4.0` or maybe `pyro-1.4.1-pre0`, and continuing to increment these tags as you simultaneously develop `funsor` and `pyro`?

Reply:
I've just pinned it to a recent commit in funsor `master` for now, like NumPyro sometimes does with jax.